src/ontology/foodon-edit.owl

```diff
@@ -18460,6 +18460,125 @@ Dioscorea japonica is used for food.</obo:IAO_0000115>
+        <rdfs:label xml:lang="en">hawaiian pizza</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003931 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003931">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00003928"/>
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03311484"/>
+        <obo:IAO_0000115 xml:lang="en">A pizza made with barbeque sauce, diced barbecue chicken, cilantro, peppers, and onion all covered with cheese.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2021-08-09T13:38:11Z</dc:date>
+        <oboInOwl:hasExactSynonym xml:lang="en">bbq chicken pizza</oboInOwl:hasExactSynonym>
+        <rdfs:label xml:lang="en">barbecue chicken pizza</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003932 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003932">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00003928"/>
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03311484"/>
+        <obo:IAO_0000114 rdf:resource="http://purl.obolibrary.org/obo/IAO_0000428"/>
+        <obo:IAO_0000115 xml:lang="en">A pizza made with chicken, buffalo hot sauce, cheese, onion and cilantro as pizza toppings.</obo:IAO_0000115>
+        <obo:IAO_0000117 xml:lang="en">https://orcid.org/0000-0001-5275-8866</obo:IAO_0000117>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2021-08-09T13:50:23Z</dc:date>
+        <rdfs:label xml:lang="en">buffalo pizza</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003933 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003933">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00003928"/>
+        <obo:IAO_0000115 xml:lang="en">A pizza that is topped with a combination of four kinds of cheese melted together, with or without tomato sauce.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2021-08-09T14:22:17Z</dc:date>
+        <oboInOwl:hasExactSynonym xml:lang="en">4 cheese pizza</oboInOwl:hasExactSynonym>
+        <rdfs:label xml:lang="en">four cheese pizza</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003934 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003934">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00003928"/>
+        <obo:IAO_0000115 xml:lang="en">A pizza with pesto sauce, which is made from basil, pine nuts, oil, garlic and Parmesan cheese, along with other pizza toppings.</obo:IAO_0000115>
```

Comment:
Don't define pesto sauce here.  It already has its entry: http://purl.obolibrary.org/obo/FOODON_03310218